html-and-web/html-and-the-web_div-salad/index.html

```diff
@@ -0,0 +1,71 @@
+<!--
+This page looks so confusing.
+It would be much better to work with semantic HTML to structure this code.
+Let's fix this mess!
+(There are hints to help you out below some elements)
+-->
+
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Basics Excercise</title>
+    <meta charset="UTF-8" />
+  </head>
+  <!--someone forgot to wrap the content of our webpage with an HTML element,
+	which one was it again?-->
+  <body>
+    <header>
+      <h1>Jane Doe</h1>
+      <!--Which elements could be used to link to parts of the webpage,
+		which appear below?-->
+      <nav id="nav">
+        <a href="#about-me">About Me</a>
+        <a href="contact">Contact</a>
+        <a href="skills">Skillset</a>
+      </nav>
+    </header>
+    <!--Looks like the following part is the central content of our webpage.-->
+    <section>
+      <article id="about-me">
+        <title>About me</title>
+        <p>
+          Hi there! Welcome to my personal website. Here is gathered the most
+          important information about me. If you want to work with me or get to
+          know me, please don't hesitate to get in contact.
+        </p>
+        <!--What element could be used to combine an image with a descriptive caption?-->
+        <figcaption>
+          <img
+            src="https://source.unsplash.com/random/400x300?fruit"
+            alt="a random picture"
+          />
+          <p>
+            A random picture, because I would like to maintain my privacy and
+            let my code speak for me, instead of images.
+          </p>
+        </figcaption>
+      </article>
+
+      <section id="contact">
+        <title>Contact</title>
+        <!--When clicking on these element, nothing happens! But I want to be redirected to the other websites. Why is it not working?-->
+        <a href="https://github.com/spiced-academy">GitHub</a>
+        <a href="https://de.linkedin.com/">LinkedIn</a>
+        <a href="https://codesandbox.io/">CodeSandbox</a>
+      </section>
+
+      <section id="skills">
+        <title>Skillset</title>
+        <!--This part could use some bulletpoint to be more structured-->
+        <ul>
+          <li>HTML</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+          <li>React</li>
+          <li>Painting rainbows</li>
+          <li>Party</li>
+        </ul>
+      </section>
+    </section>
```

Comment:
is this an extra tag?

Reply:
Good catch. This was supposed to be main. resolved here [69e148f](https://github.com/emyjamalian/challenges/pull/1/commits/69e148f950a359d0276a6141a6dfa82acf5ee6a3)